### PR TITLE
[Imaging Uploader] fixed Incorrect 'Fields do not match filename' error message

### PR DESCRIPTION
**Pull request by maltheism**
### Brief summary of changes

Fixes the message dialog for an error when uploading.

### This resolves issue...

- [ ] Redmine? #

- [x] Github? #4568

### To test this change...

- [x] See issue reported.

### CODE REVIEW

**Comment by cmadjar** [comment]
@maltheism I found a fix to fix @nicolasbrossard's issue. I've sent a PR to your branch so you can integrate the changes.

@nicolasbrossard , could you test thoroughly that everything works as expected (https://github.com/maltheism/Loris/pull/3). I already did some testing on my end but we are never too sure ;)

**Comment by maltheism** [comment]
Hey @nicolasbrossard, I added the code of @cmadjar and hopefully it excludes the wrong file extensions now.

**Comment by cmadjar** [comment]
@maltheism This branch has conflicts that need to be resolved. Could you rebase your branch please? Thank you!

**Comment by zaliqarosli** [comment]
im rebasing

File: modules/imaging_uploader/jsx/UploadForm.js [hunk 1]
```diff
@@ -143,6 +142,31 @@ class UploadForm extends Component {
           hasError.visitLabel = true;
         }
 
+        this.setState({errorMessage, hasError});
+        return;
+      }
+      if (!fileName.match(properExt)) {
+        swal({
+          title: 'Invalid extension for the uploaded file!',
+          text: 'Filename extension does not match .zip, .tgz or .tar.gz ',
```
Comment (kchatpar) [inline modules/imaging_uploader/jsx/UploadForm.js]: It looks like this line is not being executed when the file extension is incorrect.

Comment (zaliqarosli) [reply modules/imaging_uploader/jsx/UploadForm.js]: @kchatpar you left this comment and then approved the review. did the swal end up working?

Comment (cmadjar) [reply modules/imaging_uploader/jsx/UploadForm.js]: @zaliqarosli `npm run compile` was not run for the first test so the comment above was due to the fact that the JS files were not compiled. He tested again after compiling the JS files and now it works.

**Pull request merged**